notes-to-self/afd-lab.py

```diff
@@ -0,0 +1,173 @@
+# A little script to experiment with AFD polling.
```

Comment:
Is it possible for the IOCP-based I/O manager implemented in this PR to run into the bug elicited by this script? If not, maybe comment why not?

Reply:
Right, this bug is why we have all this machinery to avoid ever issuing more than one IOCTL_AFD_POLL operation on the same socket at the same time :-). Updated comments in both `afd-lab.py` and `_io_windows.py`.